ebu_tt_live/bindings/validation/presentation.py

```diff
@@ -33,13 +33,17 @@ class StyledElementMixin(object):
     """
     _compatible_style_type = None
     _referenced_styles = None
+    _referenced_divs = None
     _inherited_styles = None
     _region_styles = None
     _validated_styles = None
     _inherited_region = None
     _specified_style = None
     _computed_style = None
 
+    def _semantic_collect_applicable_divs(self, dataset, parent_binding,div_type):
+       self._referenced_div = self
```

Comment:
@mm326 this one might be for you?

Reply:
that might have been some left over code to see if I could get the parent div. I've removed it locally from `presentation.py `and` init.py `and tested it and it doesn't break other tests. 
Yeah it can be removed, it's not necessary.

Reply:
great, please could you push the removal of the unnecessary code?

Reply:
I've pushed this change